src/components/30-organisms/footer/index.js

```diff
@@ -0,0 +1,279 @@
+// TODO fix that stuff
+/* eslint-disable import/no-extraneous-dependencies */
+import { CaretSvg } from '@axa-ch/materials/icons';
+import '@axa-ch/container';
+import { LitElement, html, svg, css, unsafeCSS } from 'lit-element';
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+
+class AXAFooter extends LitElement {
+  static get tagName() {
+    return 'axa-footer';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      dynamic: { type: Boolean },
+    };
+  }
+
+  constructor() {
+    super();
+    this.dynamic = false;
+    this.onItemClick = () => {}; // Only for react
+    this._accordionActiveIndex = -1;
+  }
+
+  updated() {
+    this._removeEmptyListElements();
+  }
+
+  _removeEmptyListElements() {
+    this.shadowRoot
+      .querySelectorAll('.o-footer__main-content-panel-list-item')
+      .forEach(el => {
+        const label = el.querySelector('slot').assignedNodes()[0];
+        // Second part of IF-statement is an IE11 workaround
```

Comment:
workaround for what bug? If comment explains it, the reader could learn to avoid the bug in their own code...!